File: API/com.debtcalculator.Domain/Models/UserModel.cs
namespace com.debtcalculator.Domain.Models
{
    public class UserModel
    {
        public UserModel(long id, string name, string email, long idProfile)
        {
            Id = id;
            Name = name;
            Email = email;
            IdProfile = idProfile;
        }

        public long Id { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }
        public long IdProfile { get; set; }
    }
}

File: UI/debtCalculator/src/app/shared/models/user.model.ts

export class UserModel {
  id: number;
  name: string;
  email: string;
  idProfile: number;
}

export class AddUserModel {
  name: string;
  email: string;
  cpf: string;
}

export class EditUserModel {
  id: number;
  name: string;
  email: string;
}

export class UserPaginationModel {
  data: UserModel[];
  total: number;
}

export class UserLoginModel {
  id: number;
  name: string;
  email: string;
  idProfile: number;
}

